src/fiskaltrust.Launcher/Commands/RunCommand.cs

```diff
@@ -83,7 +83,7 @@ public static async Task<int> HandleAsync(CommonOptions commonOptions, CommonPro
                     services.AddSingleton(_ => runServices.LauncherExecutablePath);
                 });
 
-            builder.WebHost.ConfigureBinding(new Uri($"http://[::1]:{commonProperties.LauncherConfiguration.LauncherPort}"), protocols: HttpProtocols.Http2);
+            builder.WebHost.ConfigureBinding(new Uri($"http://[::1]:{commonProperties.LauncherConfiguration.LauncherServiceUri}"), protocols: HttpProtocols.Http2);
```

Comment:
The ProcessHostService server is started here.
We need to use the functionality you provided in the `ConfigureKestrel` method instead of this line.